src/Relay/createMockNetworkLayer.ts

```diff
@@ -0,0 +1,13 @@
+import { IResolvers } from "graphql-tools/dist/Interfaces"
```

Comment:
Interesting -- I wonder if the `I<Name>` idiom is coming back 🤔

Reply:
Nooooooooooooooooo

(it's actually the default in tslint, we always turn it off)

Reply:
Yeah, please no; we’re not 🤖s but 👩‍🎨 & 👨‍🎨!